mongo/database.go

```diff
@@ -153,7 +153,7 @@ func (db *Database) processRunCommand(ctx context.Context, cmd interface{},
 		return nil, sess, errors.New("read preference in a transaction must be primary")
 	}
 
-	runCmdDoc, err := transformBsoncoreDocument(db.registry, cmd)
+	runCmdDoc, err := transformBsoncoreDocument(db.registry, cmd, false, "cmd")
```

Comment:
Should we add a test to `integration/database_test` for this?

Reply:
done